lib/assessment/label.py

```diff
@@ -43,6 +44,22 @@ def statically_label_student_work(self, rubric, student_code, student_id, exampl
                     )
                 )
             }
+        
+        results = {"metadata": {"agent": []},"data": []}
+    
+        for row in csv.DictReader(rubric.splitlines()):
+            if 'CFE' in row["Instructions"]:
+                cfe = CodeFeatures()
+                cfe.extract_features(student_code, row)
+                results["metadata"]["agent"] = ["code feature extractor", "static analysis"]
+                results["data"].append({"Label": cfe.assessment,
+                                        "Key Concept": row["Key Concept"],
+                                        "Observations": cfe.features,
+                                        "Reason": row[cfe.assessment] if cfe.assessment else ''
+                                        })
```

Comment:
Do we have to modify all rubrics to have an "Instructions" row, or is this backwards compatible with rubrics that don't have this?